shell/common/switches.h

```diff
@@ -261,6 +261,10 @@ DEF_SWITCH(EnableImpeller,
            "enable-impeller",
            "Enable the Impeller renderer on supported platforms. Ignored if "
            "Impeller is not supported on the platform.")
+DEF_SWITCH(ImpellerBackend,
+           "impeller-backend",
+           "Hardcodes a selection of which impeller backend to use. (ex "
```

Comment:
Change the description (and the comment in `settings.h`) to reflect that this is modifiable and not hardcoded
(e.g. "Requests a particular Impeller backend on platforms that support multiple backends")

Reply:
done